src/main/java/org/kainos/ea/db/JobRoleDao.java

```diff
@@ -43,4 +45,30 @@ public List<JobRole> getJobRoles(Connection c) throws ActionFailedException {
 
     }
 
+    public int createJobRole(Connection c, JobRoleRequest jobRoleRequest) throws ActionFailedException {
+
+        // Insert employee into employee table
+        String insertEmployeeStatement = "INSERT INTO job_role (job_role_title, capability_id, band_id, job_role_sharepoint_link) "+
+                "VALUES (?,?,?,?);";
+
+        try (PreparedStatement st = c.prepareStatement(insertEmployeeStatement, Statement.RETURN_GENERATED_KEYS)) {
+            st.setString(1, jobRoleRequest.getJobRoleTitle());
+            st.setInt(2, jobRoleRequest.getCapabilityId());
+            st.setInt(3, jobRoleRequest.getBandId());
+            st.setString(4, jobRoleRequest.getSharepointLink());
+
+            st.executeUpdate();
+            ResultSet rs = st.getGeneratedKeys();
+
+            if (rs.next()) {
+                return rs.getInt(1);
+            }
+
+            throw new ActionFailedException("Could not get Job Role id");
```

Comment:
Is this getting a Role ID?

Reply:
Yes, it gets the id of the role it creates